### DISCUSSION TITLE
When Use ProjectTo  Navigation List  How to write  param where ?

### DISCUSSION THREAD

**Discussion by ByZhouhang** [LuckyPennySoftware/AutoMapper#3784] [NA]
![image](https://user-images.githubusercontent.com/43258053/143195013-f9a7f6f6-1843-4d32-a436-7dabfa58b1d8.png)

**Comment by lbargaoanu** [comment]
See [this](https://github.com/AutoMapper/AutoMapper/discussions/3713#discussioncomment-1141771) and [this](https://docs.automapper.org/en/stable/Queryable-Extensions.html#parameterization).
